usl_pipeline/usl_lib/usl_lib/writers/hdf5_writers.py

```diff
@@ -0,0 +1,26 @@
+import typing
+from typing import Dict
+
+import h5py
+import numpy
+import numpy.typing as npt
+
+
+def write_to_hdf5(
+    layers: Dict[str, npt.NDArray[numpy.float32]],
+    output_stream: typing.BinaryIO,
+    group_name: str = "features",
```

Comment:
Might not even make this a parameter to prevent folks from being able to create dictionary-of-dictionary structures with hdf5 files containing multiple groups.

Reply:
Switched to a constant.